client/dashboard/profile-wizard/steps/theme/preview.js

```diff
@@ -0,0 +1,82 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Button } from 'newspack-components';
+import { Component } from '@wordpress/element';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * WooCommerce dependencies
+ */
+import { WebPreview } from '@woocommerce/components';
+
+const devices = [
+	{
+		key: 'mobile',
+		icon: 'phone_android',
+	},
+	{
+		key: 'tablet',
+		icon: 'tablet_android',
+	},
+	{
+		key: 'desktop',
+		icon: 'desktop_windows',
+	},
+];
+
+class ThemePreview extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			device: 'desktop',
+		};
+	}
+
+	render() {
+		const { onChoose, onClose, theme } = this.props;
+		const { demo_url, slug, title } = theme;
+		const { device: currentDevice } = this.state;
+
+		return (
+			<div className="woocommerce-theme-preview">
+				<div className="woocommerce-theme-preview__toolbar">
+					<Button className="woocommerce-theme-preview__close" onClick={ onClose }>
+						<i className="material-icons-outlined">arrow_back</i>
+					</Button>
+					<div className="woocommerce-theme-preview__theme-name">
+						{ interpolateComponents( {
+							mixedString: sprintf(
+								__( '{{strong}}%s{{/strong}} developed by WooCommerce', 'woocommerce-admin' ),
```

Comment:
Minor but if we ever intend on having any Marketplace developed themes, or others not developed by us, we might want to have this label be a prop from the theme itself.

Reply:
That's a good idea.  I'll probably leave this as is for now since we would need more information from the WCCOM side.  But if we do implement something like this, maybe a `source` field in the API response would be enough.